### DISCUSSION TITLE
add total packages deployed in Nest.land

### DISCUSSION THREAD

**Discussion by SRNV** [nestdotland/nest.land#231] [Ideas]
sometimes I just want to see how many eggs are in the land
![Screenshot from 2020-07-21 10-26-07](https://user-images.githubusercontent.com/37340675/88031491-f826f380-cb3c-11ea-8730-70a351b45011.png)

**Comment by johnletey** [comment]
I completely agree with this!

**Comment by t8** [comment]
Great idea. I'm all for this as well.
